CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.2 (2019-05-17)
+
+- Fixed autocomplete component to emit a `selectionChange` event when input is cleared. [#22](https://github.com/blackbaud/skyux-lookup/issues/22)
+- Fixed autocomplete component to no longer have improper Angular form control states when initialized with a value. [#20](https://github.com/blackbaud/skyux-lookup/issues/20)
```

Comment:
Fixed the autocomplete component to remove improper Angular form control states when initialized with a value.

Does "remove" change the meaning?

Reply:
Hmm... well, it doesn't remove the states. It sets them properly. How about I use the positive tone and shorten it a bit. Something like this... 
`Fixed the autocomplete component to have proper Angular form control states when initialized with a value.`?

Reply:
Much better!

Reply:
done.